### DISCUSSION TITLE
Crashing on launch

### DISCUSSION THREAD

**Discussion by ccdffddfddfdsfedeee** [RedShyGuy/Vapecord-ACNL-Plugin#33] [Q&A]
Video: https://streamable.com/fk6gru

**Comment by SirEnder125** [comment]
I'm no pro at these things, but have you tried deleting the CTRPFData.bin?

**Reply by ccdffddfddfdsfedeee** [reply]
Still not working, same error

**Reply by SirEnder125** [reply]
Okay, do you have all the proper .txt files? ("english.txt", "japanese.txt", "color.txt", etc.)?

**Reply by ccdffddfddfdsfedeee** [reply]
Yes

**Reply by SirEnder125** [reply]
What about reinstalling the plugin?

**Reply by ccdffddfddfdsfedeee** [reply]
Still crashing

**Reply by SirEnder125** [reply]
Sorry, I do not know. I'll let someone smart help you. lol
Unless it's Luma3ds's problem... You could try reinstalling that.

**Reply by RedShyGuy** [reply]
It crashes by trying to get the player data hmmm. Thats really weird, does the game work without the plugin?

**Reply by ccdffddfddfdsfedeee** [reply]
Yes, it only crashes when it is turned on

**Reply by RedShyGuy** [reply]
Thats even weirder. Do you use the newest version of acnl and use the newest version of the plugin. Also do you use 2 plugins at the same time?

**Reply by ccdffddfddfdsfedeee** [reply]
The game and the plugin are the newest version and there is only one for the game

**Reply by RedShyGuy** [reply]
Do other cheat plugins work?

**Reply by RedShyGuy** [reply]
And can you show me the contents of your plugin folder?

**Reply by ccdffddfddfdsfedeee** [reply]
Plugins for other games work, and 
[this is the contents of the folder](https://user-images.githubusercontent.com/61155247/105999271-7c71aa80-607b-11eb-802d-4dd3cba52360.png)

**Reply by RedShyGuy** [reply]
Try to delete the plugin.bin
Though I am really not sure what exactly causes the problem

**Reply by ccdffddfddfdsfedeee** [reply]
Tried it and it went to the setup but still crashes at the same place

**Reply by ccdffddfddfdsfedeee** [reply]
Any help?

**Reply by RedShyGuy** [reply]
I am sorry I really don't know what the issue is😕

**Reply by ccdffddfddfdsfedeee** [reply]
It is working now. Is it possible to get banned when going on the online island?

**Reply by RedShyGuy** [reply]
A possibility is always there. But no one ever got banned since the games release for hacking online in new leaf. So you dont have to worry about that.
